charts/flyte-core/values-eks.yaml

```diff
@@ -302,7 +302,7 @@ configmap:
 
   logger:
     logger:
-      level: 5
+      level: 3
```

Comment:
Can we just drop this? the default is good enough, right?

Reply:
oh good point! done